integrations/ad_integration/tests/test_ad_sync.py

```diff
@@ -0,0 +1,661 @@
+# TODO: Fix imports in module
+import sys
```

Comment:
Voldsom mængde tests. Bliver dyrt at vedligeholde og sætte sig ind i for den næste maintainer.

Reply:
Jeg har det modsatte syn på det.

Ved at have mange tests og en dækkende test suite, så behøves en ny maintainer ikke sætte sig ind i hele kode basen for at ændre ting (og for at forstå konsekvenserne af en given ændring), man kan derimod bare begynde at arbejde i et hjørne af kodebasen, og når så man ser tests fejle, kan man læse og forstå de specifikke tests der fejler, og dermed få indsigt i den del af koden man har påvirket.

Reply:
Det er også korrekt.
Men det behøver han heller ikke for at rette den ældre kode. Blot sætte sig ind i det sted, man skal rette. Rette - og teste med kunden som endelig godkender.